modules/azure/src/main/java/org/testcontainers/containers/CosmosDBEmulatorContainer.java

```diff
@@ -0,0 +1,98 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.KeyStore;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.testcontainers.utility.DockerImageName.parse;
+
+/**
+ * An Azure CosmosDB container
+ *
+ * @author Onur Kagan Ozcan
```

Comment:
could you please omit `@author` tag here and in other places? We do not use them (although there are a few that we missed) and instead rely on Git history. Thanks!

Reply:
Of course 👍